runtime/doc/lua.txt

```diff
@@ -4308,4 +4308,30 @@ vim.text.hexencode({str})                               *vim.text.hexencode()*
     Return: ~
         (`string`) Hex encoded string
 
+
+==============================================================================
+Lua module: tohtml                                                    *tohtml*
+
+
+:TOhtml {file}                                                       *:TOhtml*
+Generates HTML and writes to a file and opens it in a new split window. If the
+file is not given then it uses a temporary file.
+
+tohtml({winid}, {opt})                                       *tohtml.tohtml()*
+    Converts a window to HTML and returns the output as a list of string.
+
+    Parameters: ~
+      • {winid}  (`integer?`) Defaults to current window.
+      • {opt}    (`table?`) Optional parameters.
+                 • title (string): Title of the html (defaults to buffer name)
+                 • number_lines (boolean): Show line numbers (defaults to
+                   `false`)
+                 • font (string|string[]): Fonts to use (defaults to
+                   `guifont`)
+                 • width (integer): Width of the html (defaults to 'textwidth'
+                   or window width)
```

Comment:
```suggestion
                 • width (integer): Width to set in the html metadata (defaults to 'textwidth'
                   or window width)
```
which of the two? either if only one is set? which takes precedence if both are set?